Test/git-issues/git-issue-981.dfy

```diff
@@ -0,0 +1,93 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+module K {
+  module KK {
+    module KKK {
+      const k := 10;
+    }
+  }
+}
+
+// Checks that refines does resolve a qualified ID
+// and does not use a local module
+module L refines K.KK.KKK {
```

Comment:
At some point, we discussed not allowed qualified names after `refines`. Is that restriction coming in a further PR or did you figure out a good way to deal with these?

Reply:
I implemented this. I could not come up with a scenario that causes trouble for qualified names vs only sibling modules. The dependencies work out fine.

Reply:
Documented the potential problem in #994, for the next round of work.